nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/test/groovy/org/apache/nifi/processors/script/ExecuteScriptGroovyTest.groovy

```diff
@@ -16,23 +16,20 @@
  */
 package org.apache.nifi.processors.script
 
+
 import org.apache.nifi.script.ScriptingComponentUtils
 import org.apache.nifi.util.MockFlowFile
 import org.apache.nifi.util.StopWatch
 import org.apache.nifi.util.TestRunners
-import org.junit.After
-import org.junit.Before
-import org.junit.BeforeClass
-import org.junit.Ignore
-import org.junit.Test
+import org.junit.*
 import org.junit.runner.RunWith
 import org.junit.runners.JUnit4
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 
 import java.util.concurrent.TimeUnit
 
-import static org.junit.Assert.assertNotNull
+import static org.junit.Assert.*
```

Comment:
We don't have CheckStyle rules being applied to Groovy code at the moment, but we discourage the use of star imports (and even prohibit them in Java code via a CheckStyle rule), so would be good to have individual imports here.

Reply:
done